waku/waku_rln_relay/group_manager/static/group_manager.nim

```diff
@@ -66,20 +66,17 @@ method register*(
 ): Future[void] {.async: (raises: [Exception]).} =
   initializedGuard(g)
 
-  await g.registerBatch(@[rateCommitment])
+  let leaf = rateCommitment.toLeaf().get()
```

Comment:
I agree, however, since the static group manager uses the [`StaticGroupKeys`](https://github.com/waku-org/nwaku/blob/66eba029efc9d97d7ba224807e414d346cdace6e/waku/waku_rln_relay/constants.nim#L45) we can be sure that this wont error :) didn't want to be too verbose where it isn't required. Can change if you insist though